db/schema.rb

```diff
@@ -30,15 +30,15 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "vsphere_root_folder", default: ""
-    t.string "puppet_init_path", default: "/"
```

Comment:
This PR doesn't include migrations, no changes to schema.rb are needed

Reply:
Fix this and merge! :1st_place_medal:

Reply:
https://github.com/hpi-swt2/vm-portal/pull/505/commits/6b794835ec9332fd902e8cbf899d86db4bad2e34